### ISSUE TITLE
Improve accessbility across SRCF online presence

### ISSUE THREAD

**Issue by matiasilva** [SRCF/projects#5]
## Project/idea summary

Aim to be [(WCAG) 2.0](https://www.w3.org/TR/WCAG20/) compliant across our website, docs, blog and control panel.

## Motivation

There are many reasons for trying to be more accessible, and I think that's fairly self-evident. The reason that this has not been done already is that we lack familiarity with how to do this properly. Let's strive to be as inclusive as possible.
